test/integration/util/util.go

```diff
@@ -456,3 +470,488 @@ func PodScheduled(c clientset.Interface, podNamespace, podName string) wait.Cond
 		return true, nil
 	}
 }
+
+// InitDisruptionController initializes and runs a Disruption Controller to properly
+// update PodDisuptionBudget objects.
+func InitDisruptionController(t *testing.T, testCtx *TestContext) *disruption.DisruptionController {
```

Comment:
These are a lot of function that are now under the ownership of sig-testing. Are you comfortable with that?

Maybe we can have a `test/integration/scheduler/util` package?

Reply:
scheduler are the main users of the library already

```
grep -r "k8s.io/kubernetes/test/integration/util" test/integration/
test/integration/ipamperf/cloud.go:     "k8s.io/kubernetes/test/integration/util"
test/integration/ipamperf/ipam_test.go: "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/preemption_test.go:  testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/priorities_test.go:  testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/queue_test.go:       testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/scheduler_test.go:   testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/taint_test.go:       testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/extender_test.go:    testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/framework_test.go:   testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/predicates_test.go:  testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler/util/util.go:        testutils "k8s.io/kubernetes/test/integration/util"
test/integration/scheduler_perf/util.go:        "k8s.io/kubernetes/test/integration/util"
test/integration/node/lifecycle_test.go:        testutils "k8s.io/kubernetes/test/integration/util"
```
and there are already scheduler only things like

https://github.com/kubernetes/kubernetes/blob/a330d3b9721ff2a9c78554d390a134c905c32c99/test/integration/util/util.go#L384-L396


I think that is simpler to have only one util package, I don't think that we suffers of bottlenecks on reviews and approvals in the integration framework, so if you don't mind I prefer to have only one package